stock-monitor/index.js

```diff
@@ -0,0 +1,160 @@
+// To update Grist table, we first need to know what table we are connected to.
+// Here we will listen to all messages sent from Grist to our custom widget,
+// and if any of this message has a tableId, we will store it in global window
+// object.
+let tableId;
+grist.on("message", (data) => {
```

Comment:
(for later grist-widget api improvements - it would be nice if this information was available in a less hacky way)

Reply:
I added this to the backlog.